Projects/App/Sources/Global/Extension/UITableView+.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  UITableView+.swift
+//  App
+//
+//  Created by Chanhee Jeong on 2022/10/25.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+
+import UIKit
+import DesignSystem
+
+extension UITableView {
+    
+    enum EmptyViewType {
+        case search
+        case noresult
+    }
+    
+    func setEmptyView(message: String, type: EmptyViewType) {
+        
+        let emptyView = UIView(frame: CGRect(x: self.center.x,
+                                             y: self.center.y,
+                                             width: self.bounds.size.width,
+                                             height: self.bounds.size.height))
+           
+        let iconView: UIImageView = {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+            $0.image = UIImage((type == .search) ?  .img_emptyfriends_search : .img_emptyfriends_noresult)
+            return $0
+        }(UIImageView())
+        
+        let messageLabel: UILabel = {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+            $0.text = message
+            $0.textColor = (type == .noresult) ? .zestyColor(.dim) : .label
+            $0.numberOfLines = 0
+            $0.textAlignment = .center
+            $0.font = .systemFont(ofSize: 17, weight: .medium)
+            $0.sizeToFit()
+            return $0
+        }(UILabel())
+        
+        self.addSubview(emptyView)
+        
+        emptyView.addSubviews([iconView, messageLabel])
+        NSLayoutConstraint.activate([
+            iconView.centerYAnchor.constraint(equalTo: emptyView.centerYAnchor, constant: -100),
```

Comment:
p1;

제 원래 의도는 emptyView가 키보드가 올라와 있을 때 검색바와 키보드 사이 중앙에 위치해 있는거였습니다.
이렇게 고치는게 베스트이긴 한데 시간이 없으니 일단은 간격을 조금만 넓혀도 될 것 같아요!
제가 해봤는데 -180정도면 모든 기기에서 괜찮더라고요.

<img width="1177" alt="image" src="https://user-images.githubusercontent.com/75792767/197740153-8ca11416-3015-49a1-b3fb-3956e749efa5.png">

```suggestion
            iconView.centerYAnchor.constraint(equalTo: emptyView.centerYAnchor, constant: -180),
```

Reply:
반영이요!
테이블뷰의 배경을 거는 거라서 미세한 조절이 어려운데 나중에 한번 시간날때 시도해볼게요 
7da5c9ba03943de1327f512ec74e7169a90fdbc6

Reply:
사실 저는 테이블뷰는 리스트가 있을 때만 쓰고 EmptyView일 때는 테이블뷰를 안쓰는걸 생각하고 있었습니다. 가능할지는 모르겠지만?